src/Microsoft.AspNet.Mvc.Core/MvcBuilderExtensions.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.AspNet.Builder
     /// <summary>
     /// Extension methods for <see cref="IApplicationBuilder"/> to add MVC to the request execution pipeline.
     /// </summary>
-    public static class BuilderExtensions
+    public static class MvcBuilderExtensions
```

Comment:
Yeah, I double checked and you're right. We'll only need one version of `AddMvc`